library/oneview_server_profile.py

```diff
@@ -16,236 +16,260 @@
 # limitations under the License.
 ###
 
-import hpOneView as hpov
-from hpOneView.exceptions import *
 from ansible.module_utils.basic import *
-
-__author__ = 'ChakruHP'
+from hpOneView.oneview_client import OneViewClient
 
 DOCUMENTATION = '''
 ---
 module: oneview_server_profile
-short_description: Selects a server hardware automatically based on the server hardware template.
+short_description: Manage OneView Server Profile resources.
 description:
-    - Manage the servers lifecycle with OneView Server Profiles using an existing server profile template.
+    - Manage the servers lifecycle with OneView Server Profiles using an existing server profile template. On 'present'
+      state it selects a server hardware automatically based on the server profile template.
```

Comment:
To documentation could also say that the server hardware is selected automatically only when the server hardware is ommited in the arguments of the play.